README.md

```diff
@@ -1,6 +1,9 @@
 # SPECTrecon
 SPECT image reconstruction tools in Julia
 
+This repo is almost a copy of [MIRT SPECTrecon Package](https://github.com/JeffFessler/SPECTrecon.jl),
+but with several commits ahead.
+
```

Comment:
```suggestion
```
Don't make such mods to the README (or exclude them from the PR).  Everyone knows yours is a fork...